go/service/saltpack.go

```diff
@@ -99,6 +100,7 @@ func (h *SaltpackHandler) SaltpackDecrypt(ctx context.Context, arg keybase1.Salt
 
 func (h *SaltpackHandler) SaltpackEncrypt(ctx context.Context, arg keybase1.SaltpackEncryptArg) (keybase1.SaltpackEncryptResult, error) {
 	ctx = libkb.WithLogTag(ctx, "SP")
+	ctx = h.G().SaltpackSlots.Encrypt.Use(ctx, arg.SessionID)
```

Comment:
This is wrong. This is a CLI command. Slot use could move down into `frontendEncrypt` or the difference between gui and cli rpcs could be made at a file level.